### PR TITLE
handle response code when playbook dispatcher not found(perf)

### PR DESCRIPTION
**Pull request by acosferreira**
# Description
Include response code if no response foung

FIXES:  THEEDGE-3209

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

### CODE REVIEW

**Review by adeliaferreira — DISMISSED** [review]
lgtm

**Review by loadtheaccumulator — APPROVED** [review]

File: pkg/clients/playbookdispatcher/client.go [hunk 1]
```diff
@@ -77,8 +77,13 @@ func (c *Client) ExecuteDispatcher(payload DispatcherPayload) ([]Response, error
...
 	res, err := client.Do(req)
 	if err != nil {
+		code := 500
+		if res != nil {
+			code = res.StatusCode
+		}
+
 		c.log.WithFields(log.Fields{
-			"statusCode": res.StatusCode,
+			"statusCode": code,
```
Comment (ldjebran) [inline pkg/clients/playbookdispatcher/client.go:86]: think we should remove this line completely.

**Review by ldjebran — APPROVED** [review]
lgtm

**Review by adeliaferreira — APPROVED** [review]
lgtm

**Pull request merged**